exercises/practice/say/.docs/instructions.append.md

```diff
@@ -8,9 +8,6 @@ errors for out of range, we are using Rust's strong type system to limit
 input.  It is much easier to make a function deal with all valid inputs,
 rather than requiring the user of your module to handle errors.
 
-There is a -1 version of a test case, but it is commented out.
-If your function is implemented properly, the -1 test case should not compile.
-
 Adding 'and' into number text has not been implemented in test cases.
```

Comment:
There is also no more test case with `"and"` as input upstream, so let's remove this line as well.

Reply:
Dropped that one as well, thank you!